Package.swift

```diff
@@ -4,26 +4,55 @@ import PackageDescription
 let package = Package(
   name: "PovioKit",
   platforms: [
-    .iOS(.v13)
+    .iOS(.v13), .macOS(.v13)
   ],
   products: [
-    .library(name: "PovioKitCore", targets: ["PovioKitCore"]),
-    .library(name: "PovioKitNetworking", targets: ["PovioKitNetworking"]),
-    .library(name: "PovioKitPromise", targets: ["PovioKitPromise"]),
-    .library(name: "PovioKitUI", targets: ["PovioKitUI"])
+    .library(
+      name: "PovioKitCore",
+      targets: ["PovioKitCore"]
+    ),
+    .library(
+      name: "PovioKitUtilities", 
+      targets: ["PovioKitUtilities"]
+    ),
+    .library(
+      name: "PovioKitNetworking", 
+      targets: ["PovioKitNetworking"]
+    ),
+    .library(
+      name: "PovioKitPromise", 
+      targets: ["PovioKitPromise"]
+    ),
+    .library(
+      name: "PovioKitUIKit", 
+      targets: ["PovioKitUIKit"]
+    ),
+    .library(
+      name: "PovioKitSwiftUI", 
+      targets: ["PovioKitSwiftUI"]
+    ),
   ],
   dependencies: [
-    .package(url: "https://github.com/Alamofire/Alamofire", .upToNextMajor(from: "5.6.4"))
+    .package(
+      url: "https://github.com/Alamofire/Alamofire", 
+      .upToNextMajor(from: "5.6.4")
+    )
   ],
   targets: [
     .target(
       name: "PovioKitCore",
-      path: "Sources/Core"
+      path: "Sources/Core",
+      exclude: [
+        "../Networking",
+        "../PromiseKit",
+        "../Utilities",
+        "../UIKit",
+        "../SwiftUI",
```

Comment:
Do we need to explicitly exclude sources that are not included in the first place?

Reply:
Apparently not. Removed.